spec/support/router.rb

```diff
@@ -0,0 +1,21 @@
+class Router
+  def pdp_path
+    '/products/ruby-on-rails-baseball-jersey'
+  end
+
+  def admin_item_path
+    '/admin/products/ruby-on-rails-baseball-jersey/edit'
+  end
+
+  def admin_item_pictures_path
+    'admin/products/ruby-on-rails-baseball-jersey/images'
+  end
+
+  def admin_item_variants_path
+    '/admin/products/ruby-on-rails-baseball-jersey/variants'
+  end
+
+  def admin_item_properties_path
+    '/admin/products/ruby-on-rails-baseball-jersey/product_properties'
+  end
+end
```

Comment:
as we discussed it earlier in different PR it will be better to keep all those links the way it is right now and change them if needed in one file - here